concrete/src/Validation/Response.php

```diff
@@ -7,44 +7,57 @@
 class Response implements ResponseInterface
 {
 
+    /**
+     * @var bool
+     */
     protected $valid;
+
+    /**
+     * @var ErrorList
+     */
     protected $error;
 
+    /**
+     * Response constructor.
+     */
     public function __construct()
     {
         $this->setIsValid(true);
         $this->setErrorObject(new ErrorList());
     }
 
     /**
-     * @return mixed
+     * @return bool
      */
     public function isValid()
     {
         return $this->valid;
     }
 
     /**
-     * @param mixed $valid
+     * @param bool $valid
      */
     public function setIsValid($valid)
```

Comment:
Don't you need to make a type declaration to ensure this is really passed as a Boolean?
```
 public function setIsValid(bool $valid)
```